Documentation/ExtensionArchitecture/HowTo/BackendModule/_ModuleConfiguration/_AdminModuleControllerSetUpDocHeader.rst.txt

```diff
@@ -0,0 +1,28 @@
+..  Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets
+..  Extracted from T3docs\Examples\Controller\AdminModuleController
+
+..  code-block:: php
+    :caption: Class T3docs\\Examples\\Controller\\AdminModuleController
```

Comment:
Don't know, if it is needed to escape the backslashes here. In next file the backslash is not needed.

Reply:
If you use backslashes they need to be escaped. But usually we use filenames and therefore forward slashes that don't need to be escaped